utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
```

Comment:
Why do we use `uci:foreach` here? I would use `uci:get("system", "system", "latitude")`.